packages/components/src/internal/components/assay/AssayReimportHeader.test.tsx

```diff
@@ -19,15 +19,15 @@ describe('<AssayReimportHeader/>', () => {
         const component = (
             <AssayReimportHeader hasBatchProperties={true} assay={assay} replacedRunProperties={runData} />
         );
-        const tree = renderer.create(component).toJSON();
-        expect(tree).toMatchSnapshot();
+        const { container } = render(component);
+        expect(container.firstChild).toMatchSnapshot();
```

Comment:
Just curious, why is it `container.firstChild` here and not `container`? Is it some wrapping context component?

Reply:
I believe the "firstChild" part was a copilot suggestion and it worked well so I just kept it. I removed them and it looks like there is just a wrapping `div` around these, so I pushed that change:
https://github.com/LabKey/labkey-ui-components/pull/1354/commits/49b3f2133fe4af0c66253c9d7489e52dacb20363?diff=unified&w=1